samples/android/Sample.csproj

```diff
@@ -41,6 +41,7 @@
     <WarningLevel>4</WarningLevel>
     <AndroidUseSharedRuntime>false</AndroidUseSharedRuntime>
     <ConsolePause>false</ConsolePause>
+    <AndroidSupportedAbis>armeabi-v7a;x86;arm64-v8a;x86_64</AndroidSupportedAbis>

```

Comment:
this is just to make visual studio not yell at us if we deploy to x86 emulators. it's just for the sample app.